client/app/hosting/boost/BOOST.html

```diff
@@ -19,71 +19,27 @@ <h2 data-i18n-static="hosting_tab_BOOST"></h2>
-                        </thead>
-                        <tbody>
-                            <tr data-ng-repeat="historyEntry in (boostHistory.details | orderBy:'date':true) track by $index">
-                                <th scope="row" data-ng-bind="::historyEntry.accountId"></th>
-                                <td data-ng-bind="::historyEntry.offer"></td>
-                                <td data-ng-class="{center: historyEntry.boostOffer === null}">
-                                    <span data-ng-bind="::historyEntry.boostOffer"
-                                          data-ng-if="historyEntry.boostOffer !== null"></span>
-                                    <span class="label label-danger"
-                                          data-i18n-static="hosting_tab_BOOST_history_table_boost_history_disabled"
-                                          data-ng-if="historyEntry.boostOffer === null"></span>
-                                </td>
-                                <td colspan="2" data-ng-bind="::historyEntry.date | date:'medium'"></td>
-                            </tr>
-                        </tbody>
-                    </table>
-                </div>
-
-                <div class="clearfix"
-                     data-ng-if="boostHistory.ids">
-                    <div data-pagination-front
-                         data-items="boostHistory.ids"
-                         data-paginated-items="boostHistory.details"
-                         data-current-page="currentPageBoostHistory"
-                         data-nb-pages="nbPages"
-                         data-items-per-page="itemsPerPage"
-                         data-transform-item="transformItem(item)"
-                         data-on-transform-item-done="onTransformItemDone(items)"
-                         data-page-placeholder="{{tr('pagination_page', [currentPageBoostHistory, nbPages])}}"
-                         data-item-per-page-placeholder="{{tr('pagination_display')}}"
-                         data-on-page-change="loaders.boostHistory = true"
-                         data-ng-show="!loaders.boostHistory && boostHistory.details.length > 0">
-                    </div>
-                </div>
-            </div>
+            <oui-datagrid data-rows="hostingBoosts"
+                          data-row-loader="transformItem($row)">
+                <oui-column data-title="tr('hosting_tab_BOOST_history_table_header_id')" data-property="accountId">
+                    <span data-ng-bind="$row.accountId"></span>
+                </oui-column>
+                <oui-column data-title="tr('hosting_tab_BOOST_history_table_header_offer')" data-property="offer">
+                    <span data-ng-bind="$row.offer"></span>
+                </oui-column>
+                <oui-column data-title="tr('hosting_tab_BOOST_history_table_header_boost_offer')" data-property="boostOffer">
+                    <span data-ng-class="{center: $row.boostOffer === null}">
```

Comment:
text-center :)

center class shouldn't exist :p

Reply:
Why does this class exist then? I just replicated what was there before. :-|